crates/rolldown/src/module_finalizers/scope_hoisting/impl_visit_mut.rs

```diff
@@ -365,7 +365,7 @@ impl<'me, 'ast> VisitMut<'ast> for ScopeHoistingFinalizer<'me, 'ast> {
   #[allow(clippy::collapsible_else_if, clippy::too_many_lines)]
   fn visit_expression(&mut self, expr: &mut ast::Expression<'ast>) {
     if let Some(call_expr) = expr.as_call_expression_mut() {
-      if call_expr.is_global_require_call(self.scope) && !call_expr.span.is_empty() {
+      if call_expr.is_global_require_call(self.scope) && !call_expr.span.is_unspanned() {
```

Comment:
unspand means != (0, 0), however, `is_empty` means span.start == span.end`, `EnsureSpanUnique` visitor only guarantee the span not equal to `0, 0`, change to `is_unspanned` to make sure we could rewrite transformed `require`